engine/lib/upgrades/2016102600-3.0.0-db-engine-to-innodb-d636305e6aa305a5.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Elgg 3.0.0 upgrade 2016102600
+ * db-engine-to-innodb
+ *
+ * Converts table engine to innodb
+ */
+
+set_time_limit(0);
```

Comment:
could take some time (tens of minutes) on big databases, but depends on table size and CPU power of DB server